packages/airnode-node/src/requests/request.ts

```diff
@@ -1,9 +1,9 @@
 import isEmpty from 'lodash/isEmpty';
 import { ApiCall, Request, GroupedRequests, RequestStatus, Withdrawal } from '../types';
 
-export function blockedOrIgnored<T>(request: Request<T>): RequestStatus.Blocked | RequestStatus.Ignored {
+export function shouldDropAfterBlockLimit<T>(request: Request<T>): boolean {
```

Comment:
This name is confusing. I'd suggest `isRequestBlocked`.

Reply:
Fair- `shouldDrop` doesn't pertain to what the function does. I wouldn't say `isRequestBlocked` is right for what the function does either though given it operates on blocked requests and returns a bool- how about `hasExceededIgnoredBlockLimit` - this is specific and clearly returns a bool.

Reply:
I like `hasExceededIgnoredBlockLimit`